packages/react-components/react-drawer/library/src/components/OverlayDrawer/useOverlayDrawerStyles.styles.ts

```diff
@@ -44,18 +47,24 @@ export const useOverlayDrawerStyles_unstable = (state: OverlayDrawerState): Over
   const resetStyles = useDrawerResetStyles();
   const rootStyles = useDrawerRootStyles();
 
+  const isRenderingInsideContainer = !!state.mountNode;
```

Comment:
```suggestion
  const absoluteStyles  = !!state.mountNode && rootStyles.absolute;
``````

Reply:
The `isRenderingInsideContainer` var is being used in multiple places. So I don't think this suggestion applies in this case. :)